cxplat/inc/cxplat_memory.h

```diff
@@ -12,13 +12,12 @@ CXPLAT_EXTERN_C_BEGIN
 
 // Values in this enum line up with POOL_FLAGS in Windows,
 // and other values are legal.
-typedef _Enum_is_bitflag_ enum
-{
-    CXPLAT_POOL_FLAG_NONE          = 0x00000000,
+typedef _Enum_is_bitflag_ enum {
```

Comment:
FYI, we don't seem to apply this style of `{` on the same line everywhere. Just seems we are inconsistent.

Reply:
The format-code script does it automatically (per settings in .clang-format), it's not manual.